Dockerfile

```diff
@@ -3,8 +3,5 @@ FROM continuumio/miniconda3:4.5.4
 WORKDIR /workspace
 RUN mkdir assets
 
-COPY requirements.txt /workspace
-RUN pip install --upgrade pip
-RUN pip install -r requirements.txt
-
-COPY . /workspace
+COPY . .
```

Comment:
Can you please explain the purpose of this?

Reply:
Remove redundancy because we have already specified `WORKDIR`. We would also reduce one layer that incorporates one `COPY` alone.